mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -452,8 +458,109 @@ properties:
                 - pub_sub
                 - publish_findings_to_cloud_data_catalog
                 - publish_summary_to_cscc
+                - job_notification_emails
+                - deidentify
               allow_empty_object: true
               send_empty_value: true
               properties: []
               description: |
                 Publish findings of a DlpJob to Data Catalog.
+            - !ruby/object:Api::Type::NestedObject
+              name: 'jobNotificationEmails'
+              exactly_one_of:
+                - save_findings
+                - pub_sub
+                - publish_findings_to_cloud_data_catalog
+                - publish_summary_to_cscc
+                - job_notification_emails
+                - deidentify
+              allow_empty_object: true
+              send_empty_value: true
+              properties: []
```

Comment:
Just want to double check that the value of jobNotificationEmails should be {} ?

Reply:
In the [API docs](https://cloud.google.com/dlp/docs/reference/rest/v2/Action#JobNotificationEmails), there is no information about the object of `jobNotificationEmails`. Moreover, I've tried sending `jobNotificationEmails` as {} and it's working fine. I've also checked this with the API explorer and it is not suggesting any fields inside `jobNotificationEmails`.

Following is the payload I've used to configure `jobTriggers` with `jobNotificationEmails` action:
```
---[ REQUEST ]---------------------------------------
{
 "jobTrigger": {
  "description": "Description for the job_trigger created by terraform",
  "displayName": "TerraformDisplayName",
  "inspectJob": {
   "actions": [
    {
     "jobNotificationEmails": {},
     "publishFindingsToCloudDataCatalog": null,
     "publishSummaryToCscc": null
    }
   ],
   "inspectTemplateName": "sample-inspect-template",
   "storageConfig": {
    "cloudStorageOptions": {
     "fileSet": {
      "url": "gs://mybucket/directory/"
     }
    }
   }
  },
  "status": "HEALTHY",
  "triggers": [
   {
    "schedule": {
     "recurrencePeriodDuration": "86400s"
    }
   }
  ]
 }
}

---[ RESPONSE ]--------------------------------------
{
  "name": "projects/{project_name}/jobTriggers/{job_trigger_id}",
  "displayName": "TerraformDisplayName",
  "description": "Description for the job_trigger created by terraform",
  "inspectJob": {
    "storageConfig": {
      "cloudStorageOptions": {
        "fileSet": {
          "url": "gs://mybucket/directory/"
        }
      }
    },
    "inspectTemplateName": "sample-inspect-template",
    "actions": [
      {
        "jobNotificationEmails": {}
      }
    ]
  },
  "triggers": [
    {
      "schedule": {
        "recurrencePeriodDuration": "86400s"
      }
    }
  ],
  "createTime": "2023-04-13T08:22:50.200795Z",
  "updateTime": "2023-04-13T08:22:50.200795Z",
  "status": "HEALTHY"
}
```

Reply:
I cannot find the doc for this field, either. I asked the service team and am waiting for their response. Sorry for holding the PR a little bit.

Reply:
I checked with the service team and they confirmed that this field should be an empty json object.